.travis.yml

```diff
@@ -15,4 +15,5 @@ install:
   - python setup.py install
 
 script:
-  - python runtests.py
+  - python runtests.py -x lint
+  - if [[ $TRAVIS_PYTHON_VERSION == '3.6-dev' ]]; then flake8; fi
```

Comment:
Last time I checked 3.6 still wasn't a thing which is why I kept 3.6-dev. Will change and see what happens.